services/spar/spar.integration.yaml

```diff
@@ -14,6 +14,9 @@ saml:
       surname: Girlfriend
       email: email:president@evil.corp
 
+metaUrl:        https://app.wire.com/sso/metadata         # corresponds to 'APIMeta'
```

Comment:
Should be under `spInfo`.  There are at least two metadata urls, one for IdP, one for SP, and it's not clear from the name here which is meant.

Also, should the comment be in the source code?  That's what it refers to.